numpy/random/tests/test_extending.py

```diff
@@ -75,10 +76,9 @@ def test_cython(tmp_path):
     assert so1 is not None
     assert so2 is not None
     # import the so's without adding the directory to sys.path
-    from importlib.machinery import ExtensionFileLoader 
-    extending = ExtensionFileLoader('extending', so1).load_module()
-    extending_distributions = ExtensionFileLoader('extending_distributions', so2).load_module()
-
+    extending = exec_mod_from_location('extending', so1)
```

Comment:
`flake8` points out that `extending` is assigned but never used.  
```suggestion
    exec_mod_from_location('extending', so1)
```